QUEST_LV_0100.tsv

```diff
@@ -6403,101 +6403,101 @@ QUEST_LV_0100_20150317_006402	{memo X}Mr. Juan asked you to bring him Yukopus Le
-QUEST_LV_0100_20150323_006433	Death may be the price I pay for my sin. {nl}Even if it is the curse of the demons, it is still a sin I've made.
-QUEST_LV_0100_20150323_006434	I don't want to.. die like this.. {nl}Mother..
-QUEST_LV_0100_20150323_006435	Wait. Isn't the the eye of Succubus? {nl}Oh my. How did you get that.. Can I take a look? 
-QUEST_LV_0100_20150323_006436	I've heard about the eyes. {nl}Succubus envied the necklace with the blessings of Cathedral so she put it in her eyes. {nl}
-QUEST_LV_0100_20150323_006437	As you know, I may be an aspiring Elementalist? {nl}I can turn that back into a necklace. {nl}
-QUEST_LV_0100_20150323_006438	Done. {nl}Do you know about the five necklaces of Mayvern?{nl}
-QUEST_LV_0100_20150323_006439	It was created to remind people of the 5 sins. {nl}Though Naktis made it look like a curse on the Pilgrim's Way.{nl}
-QUEST_LV_0100_20150323_006440	{memo X}Looks like the necklace belongs to you. {nl}If it wasn't for you, teachings of Mayvern would have been left as a curse.
-QUEST_LV_0100_20150323_006441	$Actually, I want to go to Kvailas Forest. {nl}I want to see the source of evil with my two eyes.
-QUEST_LV_0100_20150323_006442	Thank you. The blacksmith in Klaipeda sells the tools. {nl}But before that, stones in Stele Road are known for their fine quality so I ask of your favor. 
-QUEST_LV_0100_20150323_006443	Done. Please place this in the entrance of Underground Mausoleum in Zachariel Crossroad. {nl}Will this be enough commfort.
-QUEST_LV_0100_20150323_006444	{memo X}Hmm. You're really good as they say. {nl}Rolandas is waiting for you in Akmens Ridge.
-QUEST_LV_0100_20150323_006445	{memo X}Please be careful. {nl}A dark aura similar to the voice a while ago is covering the Thron Forest. {nl}Especially the Kavilas Forest.. I have sharp ears for things like this..
-QUEST_LV_0100_20150323_006446	The Necroventer that occupied Kule Peak is gone. {nl}Kule Peak will be peaceful again.
-QUEST_LV_0100_20150323_006447	It's preparation for releasing the seal of Tiltas Vallye. {nl}The sap was collected little by little over centuries. {nl}Please handle it with care. 
-QUEST_LV_0100_20150323_006448	This is it. {nl}Bring the Release Crystal to Ramstis Ridge. {nl}Students of Ramstis will be welcome you there. 
-QUEST_LV_0100_20150323_006449	The day for the Great King's legacy finding its true owner has come. {nl}Hurry, go to the students of Gustas in Ramstis Ridge.
-QUEST_LV_0100_20150323_006450	The teacher is poisoned and can't breathe. {nl}Quickly borrow antidote from others in Akmens Ridge. 
-QUEST_LV_0100_20150323_006451	I had no idea about Rukas Plateau. {nl}What are they trying to by releasing the seal? {nl}
+QUEST_LV_0100_20150323_006422	{memo X}You need to use the cable car to go from Srautas to Leaf Veil Plateau. {nl}I heard people used to hang baskets on ropes in the past. {nl}Well the cable cars are scary enough for me. 
```

Comment:
Yup, Leaf Veil Plateau is Gele Plateau.
For Memo X, you can check the Korean file. 
If it is not yet Memo X'ed, I think it most probably will be Memo X'ed later on but it's still used in game for now. (which is the case for this line)
{memo X} is added by the system so it's best not to add them manually.